src/worker/scheduler/Scheduler.js

```diff
@@ -4,11 +4,12 @@ import { merge } from "lodash-es";
 // Internal dependencies.
 import Task from "./Task";
 
+
 const DEFAULT_CONFIGURATION = {
 	pollTime: 50,
 };
 
-class Scheduler {
+export default class Scheduler {
```

Comment:
Eslint complains about the missing JSdoc comment here.
Please don't forget to adjust the eslint warning limits after fixing! 😄

Reply:
I'm the right person to 'boy scout' this I suppose... done!